seed/views/00901_team_case_overview.js

```diff
@@ -18,9 +18,12 @@ exports.seed = function (knex, promise) {
     JOIN app.workload_owner wo ON wo.id = w.workload_owner_id
     JOIN app.offender_manager om ON om.id = wo.offender_manager_id
     JOIN app.team t ON wo.team_id = t.id
-    LEFT JOIN app.workload_points_calculations wpc ON wpc.workload_id = w.id
+    JOIN app.workload_points_calculations wpc ON wpc.workload_id = w.id
     JOIN app.workload_points wp ON wpc.workload_points_id = wp.id 
-    GROUP BY wo.id;`
+    JOIN app.workload_report wr ON wr.id = wpc.workload_report_id
+  WHERE wr.effective_from IS NOT NULL
+    AND wr.effective_to IS NULL
+  GROUP BY wo.id;`
```

Comment:
Will this script be run during migrations? I thought once a script was run once it was ignored every other time?

Reply:
So, because it is raw sql, its executed to drop the view first, then execute it each time without fail.

Reply:
`return knex.schema
    .raw('DROP VIEW IF EXISTS app.ldu_caseload_view;')
    .raw('SET ARITHABORT ON')
    .raw(sql)`

Reply:
Yeh, views are run each time regardless, as a seed rather than a migration